src/app/context/AccountContext.tsx

```diff
@@ -32,11 +26,9 @@ interface AccountContextType {
    */
   setAccountId: (id: string) => void;
   /**
-   * Fetch the additional account info: alias/balance and update account
+   * revalidation (mark the data as expired and trigger a refetch) for the resource
    */
-  fetchAccountInfo: (options?: {
-    accountId?: string;
-  }) => Promise<AccountInfo | undefined>;
+  updateAccountInfo: () => Promise<void>;
```

Comment:
I wonder if this method could be renamed to fetchAccountInfo? in my opinion it doesn't actually "update" the account